HLTrigger/Configuration/python/HLT_2023v12_cff.py

```diff
@@ -14034,11 +14034,12 @@
     pixelHitsSrc = cms.InputTag( "hltSiPixelRecHitsSoA@cuda" ),
     TopFolderName = cms.string( "SiPixelHeterogeneous/PixelRecHitsGPU" )
 )
-fragment.hltSiPixelRecHitsSoACompareGPUvsCPU = cms.EDProducer( "SiPixelPhase1CompareRecHitsSoA",
-    pixelHitsSrcCPU = cms.InputTag( "hltSiPixelRecHitsSoA@cpu" ),
-    pixelHitsSrcGPU = cms.InputTag( "hltSiPixelRecHitsSoA@cuda" ),
+fragment.hltSiPixelRecHitsSoACompareGPUvsCPU = cms.EDProducer( "SiPixelPhase1CompareRecHits",
```

Comment:
you are not supposed to touch these files directly. 
Please provide a suitable customization function in https://github.com/cms-sw/cmssw/blob/master/HLTrigger/Configuration/python/customizeHLTforCMSSW.py that modifies these configurations.
STORM will take care of migrating these files centrally.

Reply:
Thanks Marco: we're on that!
(there was a miss-communication between me and Breno).

Reply:
Hi @mmusich. Yes, this was my mistake. I was just talking to @AdrianoDee about it and he explained that I should create a customization instead of changing fragments directly. I'll remove the last commit and make the customization